test/mocks/MockDataRepository.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package mocks
+
+import base.BaseSpec
+import models.DynamicDataModel
+import org.scalamock.handlers.{CallHandler1, CallHandler2}
+import org.scalamock.scalatest.MockFactory
+import play.api.libs.json.Json.JsValueWrapper
+import reactivemongo.api.commands.{DefaultWriteResult, WriteError, WriteResult}
+import repositories.DynamicStubDataRepository
+
```

Comment:
Does there need to be a gap between these imports? Not sure if there is a particular reason for this?

Reply:
IntelliJ adds gaps between scala (and I think java) imports and other types of imports by default. I think you can update this behaviour in IntelliJ settings somewhere but seeing as most people have this enabled and you see it all over HMRC repos I personally gave up trying to change that a while ago to keep it consistent.